packages/story-editor/src/components/floatingMenu/elements/mute.js

```diff
@@ -24,13 +24,25 @@ import { __ } from '@googleforcreators/i18n';
  * Internal dependencies
  */
 import { IconButton } from './shared';
+import useVideoTranscoding from './shared/useVideoTranscoding';
 
 function Mute() {
+  const { canMute, isMuting, isDisabled, handleMute } = useVideoTranscoding();
+
+  if (!canMute) {
+    return null;
+  }
+
+  const title = isMuting
+    ? __('Removing audio', 'web-stories')
+    : __('Remove audio', 'web-stories');
+
   return (
     <IconButton
       Icon={Icons.Muted}
-      title={__('mute video', 'web-stories')}
-      onClick={() => {}}
+      title={title}
+      isDisabled={isDisabled || isMuting}
```

Comment:
Hm, this is not related to this PR but in case of some videos, I get this console error:
<img width="1378" alt="Screenshot 2022-03-03 at 11 29 22" src="https://user-images.githubusercontent.com/3294597/156536352-3379268a-110c-4e41-a3fb-6df1a35bb31e.png">

Unfortunately I'm not able to repro it consistently. @spacedmonkey is this error something you've seen recently?

Reply:
I was not able to replicate that. Can provide steps to replicate or a video or something.

Reply:
@miina Perhaps the original unmuted video got deleted in the meantime?

Reply:
@swissspidy Hm, I thought that issue got fixed somewhere 🤔 Can't find it currently though for muting, saw just this for trimming: https://github.com/GoogleForCreators/web-stories-wp/issues/10245

Reply:
Hmm, this seems unrelated to this PR. I'm not seeing any issues. I fixed the original comment here in 0c0d1a483a and will mark as resolved for now.